.github/workflows/ci.yml

```diff
@@ -11,6 +11,9 @@ jobs:
     name: Pod Lint
     runs-on: macOS-11
     steps:
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.7.6'
```

Comment:
I'd like to use a newer version of ruby, but this one works and it's the latest supported by the `setup-ruby` script